src/main/java/org/isf/menu/gui/UserBrowsing.java

```diff
@@ -278,10 +262,18 @@ public void ancestorAdded(AncestorEvent event) {
 				int answer = MessageDialog.yesNo(null, "angal.userbrowser.deleteuser.fmt.msg", selectedUser.getUserName());
 				try {
 					if (answer == JOptionPane.YES_OPTION) {
-						userBrowsingManager.deleteUser(selectedUser);
-						userList.remove(table.getSelectedRow());
-						model.fireTableDataChanged();
-						table.updateUI();
+						try {
+							userBrowsingManager.deleteUser(selectedUser);
+							userList.remove(table.getSelectedRow());
+							model.fireTableDataChanged();
+							table.updateUI();
+						} catch (OHServiceException ex) {
+							selectedUser.setDeleted(true);
+							userBrowsingManager.deleteUser(selectedUser);
+							userList.set(table.getSelectedRow(), selectedUser);
+							model.fireTableDataChanged();
+							table.updateUI();
+						}
```

Comment:
I would show a MessageDialog.yesNo message reporting why it is not possible to delete the user and asking if to proceed with marking as deleted

Reply:
or also without asking, just informing. But in the latter case I would also provide a way to "undelete" a user, probably just allowing to edit the user (you disabled this option) and uncheck that field.

Reply:
and I think UserGroups should follow the same logic as Users